pkgs/development/compilers/rust/rustc.nix

```diff
@@ -97,11 +97,11 @@ stdenv.mkDerivation {
     rm -vr src/test/run-make/linker-output-non-utf8 || true
 
     # Remove test targeted at LLVM 3.9 - https://github.com/rust-lang/rust/issues/36835
-    rm -vr src/test/run-pass/issue-36023.rs || true
+    rm -fvr src/test/run-pass/issue-36023.rs
```

Comment:
Is there a chance we can get the `|| true` version? To me it feels more explicit like "Try to remove it but we don't care if it fails" which is what these lines are supposed to do. Same in line 104.